corehq/apps/app_manager/templates/app_manager/partials/build_errors.html

```diff
@@ -195,6 +195,11 @@ <h4 class="alert-heading">
               {% blocktrans with module_name=error.module.name|trans:langs %}
               The case list in <a href="{{ module_url }}">{{ module_name }}</a> can not use the same "search input instance name" as its Parent Select Menu.
               {% endblocktrans %}
+            {% case "search on clear with auto select" %}
```

Comment:
add test for this case

Reply:
[b52eced](https://github.com/dimagi/commcare-hq/pull/33983/commits/b52eced512c8c1dcbdcb918df790aa6e5e0f607d)